### PR TITLE
Fixes #881 Remove outdated commands in documentation

### LINKED ISSUE #881: Change documentation to support current version of Django
## Description
For the current version of Django . the following commands are outdated and are now replaced by a single command of migrate.They need to be removed from the documentation to allow users to install the app with ease.
`python manage.py sqlall app_name_here`
`python manage.py syncdb`

## Mocks
screenshot from 2018-11-11 07-15-20

### Update [Required]
- [ ] Documentation needs to remove these commands.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by Tlazypanda**
# Description
Outdated commands are removed and replaced with the correct command as mentioned in the issue.

Fixes #881 

# Type of Change:
**Delete irrelevant options.**
- Documentation

# Checklist:
**Delete irrelevant options.**

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@Monal5031 Check it out whenever free :smile:

File: docs/Installation_Guide.md [hunk 1]
```diff
@@ -212,13 +212,8 @@ Make sure to exit the postgres client before proceeding to the next steps:
...
 Change directory to where you can find the **manage.py** file (this is located in the top level directory for the project). If you are installing the project on the VM, the project will be located within the **/vagrant/vms** directory.
 
-To view the sql commands that will be generated from syncdb, run the command:
-
-    python manage.py sqlall app_name_here
-
-To generate the database tables that correspond to the Django models, run the command:
-
-    python manage.py syncdb
+Run the following command:
```
Comment (Monal5031) [inline docs/Installation_Guide.md]: To generate the database tables

**Comment by Tlazypanda** [comment]
@Monal5031 Have made the changes :smile:

File: docs/Installation_Guide.md [hunk 2]
```diff
@@ -212,13 +212,9 @@ Make sure to exit the postgres client before proceeding to the next steps:
...
-To view the sql commands that will be generated from syncdb, run the command:
-
-    python manage.py sqlall app_name_here
-
-To generate the database tables that correspond to the Django models, run the command:
-
-    python manage.py syncdb
+To generate the database tables 
+Run the following command:
+    python manage.py migrate
```
Comment (Monal5031) [inline docs/Installation_Guide.md]: Md needs an extra empty line to display a newline, also put this command in codeticks (`).

File: docs/Installation_Guide.md (hunk 1 above)
Comment (Monal5031) [reply docs/Installation_Guide.md]: @Tlazypanda single sentence please.

File: docs/Installation_Guide.md (hunk 2 above)
Comment (Monal5031) [reply docs/Installation_Guide.md]: add a newline in between and put the command in backtick (`)
@Tlazypanda

Comment (Tlazypanda) [reply docs/Installation_Guide.md]: Done @Monal5031

Comment (Monal5031) [reply docs/Installation_Guide.md]: codeticks?

File: docs/Installation_Guide.md [hunk 3]
```diff
@@ -212,13 +212,11 @@ Make sure to exit the postgres client before proceeding to the next steps:
...
-    python manage.py sqlall app_name_here
+To generate the database tables run the following command:
 
-To generate the database tables that correspond to the Django models, run the command:
 
-    python manage.py syncdb
+    python manage.py migrate
 
 NOTE: In case, you get the following error django.db.utils.ProgrammingError: relation "auth_user" does not exist, while running the above command do the following
     python manage.py migrate auth
```
Comment (Monal5031) [inline docs/Installation_Guide.md]: Same here, add a new line and put it in code block, also remove the line `Now again try running the command python manage.py syncdb`

**Comment by Monal5031** [comment]
@Tlazypanda Your changes look good, just need a rebase of branch on top of develop, because your changes in the commit include changes from previous commit too [see here](https://github.com/systers/vms/commit/c07d254962a9bdfc5769bf225ef6ea0fc4897287#diff-b9dfbd6629c62d2d0cb2e27a0c3daec3R256)

**Comment by Tlazypanda** [comment]
@Monal5031 Can you review it now? :sweat_smile:

**Comment by Monal5031** [comment]
@Tlazypanda Is your branch `b` rebased on top of systers/develop? I'm seeing changes from your previous commits too. [check here](https://github.com/systers/vms/commit/c07d254962a9bdfc5769bf225ef6ea0fc4897287)

**Pull request merged**